go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/strongswan/govici
```

Comment:
Drop this as well please. With the release of Go 1.13, I do want to add the module support, but I'd prefer it in its own commit.